packages/amf/amf.0.1.2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis: "Parser/serializer for Adobe's Action Message Format"
+maintainer: "Brian Caine <brian.d.caine@gmail.com>"
+authors: "Brian Caine <brian.d.caine@gmail.com>"
+license: "LGPL with OCaml linking exception"
+homepage: "https://briancaine.github.io/ocaml-amf/"
+bug-reports: "https://github.com/briancaine/ocaml-amf/issues"
+depends: [
+  "ocaml" {> "4.03"}
+  "dune" {>= "1.0"}
+  "ppx_deriving"
+  "ppx_sexp_conv"
+  "base-threads"
+  "core" {>= "v0.9.1"}
+  "stdint"
+  "sexplib"
+  "bisect_ppx" {build}
```

Comment:
```suggestion
  "bisect_ppx" {with-test}
```
Same question for this one, though I'm not sure what it does.

Reply:
https://github.com/aantron/bisect_ppx

Probably only needed for tests.

Reply:
Apparently it is not only needed for tests, sorry :( (but ounit probably is though)
```
#=== ERROR while compiling amf.0.1.2 ==========================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/briancaine/ocaml-amf/archive/v0.1.2.tar.gz)
# path                 ~/.opam/4.08/.opam-switch/build/amf.0.1.2
# command              ~/.opam/4.08/bin/dune build -p amf -j 2
# exit-code            1
# env-file             ~/.opam/log/amf-15-fe680d.env
# output-file          ~/.opam/log/amf-15-fe680d.out
### output ###
# File "src/amf/dune", line 9, characters 50-60:
# 9 |   (pps ppx_deriving ppx_sexp_conv ppx_fields_conv bisect_ppx -- -conditional)))
#                                                       ^^^^^^^^^^
# Error: Library "bisect_ppx" not found.
# Hint: try: dune external-lib-deps --missing -p amf -j 2 @@default
```